mathesar/api/serializers/records.py

```diff
@@ -53,6 +53,11 @@ def update(self, instance, validated_data):
                     e,
                     status_code=status.HTTP_400_BAD_REQUEST
                 )
+            elif type(e.orig) == ExclusionViolation:
```

Comment:
```suggestion
            elif type(e.orig) is ExclusionViolation:
```

Reply:
thanks @Aritra8438 for pointing it out , i have pushed the changes pls mark as resolved